errors_plan9.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2014-2015 Docker Inc & Go Authors. All rights reserved.
+// Copyright (C) 2017-2025 SUSE LLC. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package securejoin
+
+// ErrLoop is returned when too many symbolic links are encountered.
+var ErrLoop = errors.New("Too many symbolic links were encountered in translating the pathname")
```

Comment:
This error message is way too long, you can just copy the one used for ELOOP.

Reply:
actually, I grabbed that right out of the include file. I'll find a shorter one.